tools/editor/asset_library_editor_plugin.h

```diff
@@ -278,8 +286,8 @@ class AddonEditorPlugin : public EditorPlugin {
 	//virtual Dictionary get_state() const;
 	//virtual void set_state(const Dictionary& p_state);
 
-	AddonEditorPlugin(EditorNode *p_node);
-	~AddonEditorPlugin();
+	AssetLibraryEditorPlugin(EditorNode *p_node);
```

Comment:
It would probably be easier to review if you had split the renaming in a first commit, and the additions in another one :)